src/UnitTests/Projection/ProjectSubqueryWithParameterAsMethodArgument.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AutoMapper.UnitTests.Projection;
+public class ProjectSubqueryWithParameterAsMethodArgument
+{
+    private MapperConfiguration _config;
+
+    public static string MyConversionFunction(int value) {
```

Comment:
This is client evaluation. We don't support that with `ProjectTo`.

Reply:
Client evaluation after projection, I thought you supported everything a regular Select does.
Something like:
```
DbSet<MyEntity>().Select(e => new MyEntityDto{
    // ...
    MyField = CustomFunc(e.MyField),
    // ...
});
```
This is a valid Queryable

Reply:
No, `ProjectTo` is a way to write SQL queries.